src/main/java/randoop/operation/TypedClassOperation.java

```diff
@@ -68,12 +70,11 @@ public int hashCode() {
    */
   @Override
   public int compareTo(TypedOperation op) {
-    int result = 0;
     if (op instanceof TypedTermOperation) {
       return 1;
     }
     TypedClassOperation other = (TypedClassOperation) op;
-    result = declaringType.compareTo(other.declaringType);
+    int result = declaringType.compareTo(other.declaringType);
```

Comment:
On line 85 below, it's confusing that the overview differs from the `@return` clause.  What does it mean for an operation to represent a value, and how can a TypedClassOperation do so?